### PR TITLE
Feature/bodcdmsys 2118

### PR DESCRIPTION
**Pull request by thogar-computer**
Looks like the update to the function distance function worked

### CODE REVIEW

File: coast/COAsT.py [hunk 1]
```diff
@@ -4,9 +4,13 @@
 from dask.distributed import Client
 
 
+def setup_dask_clinet(workers=2, threads=2, memory_limit_per_worker='2GB'):
```
Comment (matt-caz) [inline coast/COAsT.py:7]: This creates a Client object but doesn't do anything with it?

File: coast/DOMAIN.py [hunk 2]
```diff
@@ -80,3 +80,33 @@ def set_command_variables(self):
...
+        # nemo_dist = xa.DataArray(nemo_dist.data.reshape(self.dataset.nav_lat.shape), dims=['y', 'x'])
+
+        # Get boolean array where the distance is less than the specified radius
+        # using np.where
+        nemo_indices_bool = nemo_dist < radius
+        nemo_indices = np.where(nemo_indices_bool.compute())
+
+        # Then these output tuples can be separated into x and y arrays if necessary.
+
+        return nemo_indices
```
Comment (matt-caz) [inline coast/DOMAIN.py:112]: Python modules should finish with an empty new-line.

File: coast/COAsT.py (hunk 1 above)
Comment (thogar-computer) [reply coast/COAsT.py:7]: No - so there are some issues with the daks client and PyCharm - we don't need to access the client currently so this is more of a future helping method. 

The client is also somewhat detached from dask, as in you don't need to pass it around once build - in fact not calling the client only cause dask to use all available resources to hand ;)

File: coast/DOMAIN.py (hunk 2 above)
Comment (thogar-computer) [reply coast/DOMAIN.py]: added in new line

**Pull request merged**